server/controller/recipe.controller.ts

```diff
@@ -0,0 +1,32 @@
+// Package imports
+import { Request, Response } from 'express';
+
+// Local imports
+import db from '../db.json';
+import IRecipe from '../interfaces/Recipe.interface';
+import sample from '../utils/sampleFromArray';
+import recipeHasKeywords from '../utils/recipeHasKeywords';
+import recipeHasNoAllergens from '../utils/recipeHasNoAllergens';
+
+export function getRecipe (req: Request, res: Response) {
+  const string: string = req.headers.string as string;
```

Comment:
can you rename this variable to something else. the word string is used in this line 4 times